### PR TITLE
add back blog

### LINKED ISSUE #202: add a readme to new docs
yarn run docs:dev
pnpm run docs:dev
npm run docs:dev

### LINKED ISSUE #203: Add the blog back to the docs
- [x] add blog
- [x] Add the blog-post specific graphic to the frontmatter of blog entries: https://vitepress.dev/reference/frontmatter-config#head (if it is possible)

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

Resolves #203 
Resolves #202 

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

**Comment by jcstein** [comment]
hm, it's working locally so will have to look into this

File: .vitepress/config.ts [hunk 1]
```diff
@@ -4,202 +4,173 @@ const telegramSVG = ` <svg width="24" height="24" viewBox="0 0 24 24" fill="none
...
 </svg>`;
 
-const { BASE: base = "/" } = process.env;
```
Comment (MSevey) [inline .vitepress/config.ts:7]: revert this so that we don't undo the update.

File: .vitepress/config.ts [hunk 2]
```diff
@@ -4,202 +4,173 @@ const telegramSVG = ` <svg width="24" height="24" viewBox="0 0 24 24" fill="none
...
 // https://vitepress.dev/reference/site-config
 export default withMermaid({
-  lang: "en-US",
+  lang: 'en-US',
   title: "Rollkit",
   description: "The open modular framework for sovereign rollups.",
   lastUpdated: true,
   cleanUrls: true,
   ignoreDeadLinks: true,
-  base: base,
```
Comment (MSevey) [inline .vitepress/config.ts:17]: and this

File: .vitepress/config.ts (hunk 1 above)
Comment (jcstein) [reply .vitepress/config.ts]: done! good catch

**Review by MSevey — APPROVED** [review]

**Pull request merged**